### PR TITLE
Chore: General notification reformatting

### PR DESCRIPTION
**Pull request by CommanderStorm**
⚠️⚠️⚠️ Since we do not accept all types of pull requests and do not want to waste your time. Please be sure that you have read pull request rules:
https://github.com/louislam/uptime-kuma/blob/master/CONTRIBUTING.md#can-i-create-a-pull-request-for-uptime-kuma

Tick the checkbox if you understand [x]: 
- [x] I have read and understand the pull request rules.

# Description
During the Review of #3176 I noticed that the Notification Providers are written in diverse styles.
This means, that reviewing them is sometimes less rigorous in terms of reviewing code quality.

Generally, the PRs for Providers this project gets are based upon the existing codebase and do not reinvent the wheel.
⇒ establishing a common formatting aids reviewability.

I have merged these four things, as I think they are related and don't impact each other much.
If preferred, I can break them out of this PR/ drop the commits.

Now to the changes I am proposing:
- I unified where in file the name of `NotificationProvider.name` is placed
- I made sure that all the providers adhere to the signature of `NotificationProvider.send()`
- I made sure that all the providers use `okMsg` if returning success messages directly from the function.
  Here a discussion should be had:
  Should this be refactored into a constant of `NotificationProvider`? I could imagine that `NotificationProvider.SENDING_SUCCESSFULL`  could be a suitable alternative.
- I made sure all providers have the URL they `POST`/`GET` to be extraced into a variable.
  => refactored this way due to Nelsons suggestion

## Does this violate the community guidelines?

I know that this PR may go against these two rules from the community guidelines:
- Existing logic is completely modified or deleted for no reason
  ⇒ See above discussion if we should extract `NotificationProvider.SENDING_SUCCESSFULL` or `okMsg`

- Unnecessary large code changes (Hard to review, causes code conflicts to other pull requests)
  ⇒ I think that these changes being in one PR makes it easier to review than if :
  - being split by proposed change
  - being split by touched module

## Type of change

Please delete any options that are not relevant.

- Other

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I ran ESLint and other linters for modified files
- [x] I have performed a self-review of my own code and tested it
- [x] I have commented my code, particularly in hard-to-understand areas
  (including JSDoc for methods)
- [x] My changes generate no new warnings
- [ ] My code needed automated testing. I have added them (this is optional task)

### CODE REVIEW

**Comment by chakflying** [comment]
All quite reasonable changes.

For the duplicate API urls, I think factoring them out to a `const` would be cleaner and easier for future changes.

**Comment by CommanderStorm** [comment]
> For the duplicate API urls, I think factoring them out to a `const` would be cleaner and easier for future changes.

Reasonable suggestion.
I will do so, after I am done with this exam phase (⇒ in 2 weeks)

**Comment by CommanderStorm** [comment]
> For the duplicate API urls, I think factoring them out to a `const` would be cleaner and easier for future changes.

I have applied this to every api which does not have a configurable url

**Comment by chakflying** [comment]
eslint:
```
Identifier 'url_v1' is not in camel case
```

Maybe change the variable name or update the lint rules.

**Comment by CommanderStorm** [comment]
Not running ESLint/... in GitHub actions too is likely something we should fix.

**Review by chakflying — APPROVED** [review]

File: server/notification-providers/lunasea.js [hunk 1]
```diff
@@ -3,25 +3,19 @@ const axios = require("axios");
...
 
         try {
+            const target = this.getTarget(notification);
             if (heartbeatJSON == null) {
                 let testdata = {
                     "title": "Uptime Kuma Alert",
                     "body": msg,
                 };
-                await axios.post(lunaseaurl, testdata);
+                await axios.post(`https://notify.lunasea.app/v1/custom/${target}`, testdata);
```
Comment (CommanderStorm) [inline server/notification-providers/lunasea.js:18]:
```suggestion
                await axios.post(`${url}/custom/${target}`, testdata);
```

File: server/notification-providers/lunasea.js [hunk 2]
```diff
@@ -3,25 +3,19 @@ const axios = require("axios");
...
 
     async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
-        let okMsg = "Sent Successfully.";
-        let lunaseaurl = "";
-        if (notification.lunaseaTarget === "user") {
-            lunaseaurl = "https://notify.lunasea.app/v1/custom/user/" + notification.lunaseaUserID;
-        } else {
-            lunaseaurl = "https://notify.lunasea.app/v1/custom/device/" + notification.lunaseaDevice;
-        }
+        const okMsg = "Sent Successfully.";
```
Comment (CommanderStorm) [inline server/notification-providers/lunasea.js:9]:
```suggestion
        const okMsg = "Sent Successfully.";
        const url = "https://notify.lunasea.app/v1";
```

File: server/notification-providers/lunasea.js [hunk 3]
```diff
@@ -32,7 +26,7 @@ class LunaSea extends NotificationProvider {
                         heartbeatJSON["msg"] +
                         `\nTime (${heartbeatJSON["timezone"]}): ${heartbeatJSON["localDateTime"]}`
                 };
-                await axios.post(lunaseaurl, downdata);
+                await axios.post(`https://notify.lunasea.app/v1/custom/${target}`, downdata);
```
Comment (CommanderStorm) [inline server/notification-providers/lunasea.js:29]:
```suggestion
                await axios.post(`${url}/custom/${target}`, downdata);
```

File: server/notification-providers/lunasea.js [hunk 4]
```diff
@@ -43,13 +37,25 @@ class LunaSea extends NotificationProvider {
                         heartbeatJSON["msg"] +
                         `\nTime (${heartbeatJSON["timezone"]}): ${heartbeatJSON["localDateTime"]}`
                 };
-                await axios.post(lunaseaurl, updata);
+                await axios.post(`https://notify.lunasea.app/v1/custom/${target}`, updata);
```
Comment (CommanderStorm) [inline server/notification-providers/lunasea.js:40]:
```suggestion
                await axios.post(`${url}/custom/${target}`, updata);
```

**Comment by CommanderStorm** [comment]
While this would mean that I am merging my PR here, I think given how simple the changes are this cannot be classified as controversial.

All changes in this PR are small and uncontroversial
⇒ merging with junior maintainer approval

**Pull request merged**